app/src/app/[locale]/(public)/properties/[id]/_components/schedule-form.tsx

```
@@ -107,6 +107,17 @@
     return property.price * intervals.length;
   }, [dateRange, property.price]);
 
+  const [isSaveDisabled, setIsSaveDisabled] = useState(true);
+
+  const handleDateChange = (dates: Object[]) => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)